src/sidebar.js

```diff
@@ -507,6 +509,12 @@ module.exports = {
       label: 'Reference',
       collapsed: false,
       items: ['reference/api']
+    },
+    {
+      type: 'category',
+      label: 'Further Reading',
+      collapsed: false,
+      items: ['further-reading/http-cookies-explained']
```

Comment:
I'd rather have this cookies information closer to the security model to make it more visible. Putting it into the "further reading" category is IMO equal to making it invisible :D

Reply:
ok that makes sense to me